src/frontend/js/widgets/Dashboard/components/DashboardItem/DashboardItemCourseEnrolling.tsx

```diff
@@ -41,15 +41,31 @@ const messages = defineMessages({
     description: 'Button to access course when the user is enrolled',
     defaultMessage: 'Access to course',
   },
-  runPeriod: {
-    id: 'components.DashboardItemEnrollment.runPeriod',
-    description: 'Text to display the period of a course run',
-    defaultMessage: 'From {startDate} to {endDate}',
+  onGoingRunPeriod: {
+    id: 'components.DashboardItemEnrollment.onGoingRunPeriod',
+    description: 'Text to display the period of an on going course run',
+    defaultMessage: 'This session started on {startDate} and will end on {endDate}',
   },
-  enrolledRunPeriod: {
-    id: 'components.DashboardItemEnrollment.enrolledRunPeriod',
-    description: 'Text to display the period of a course run',
-    defaultMessage: 'You are enrolled for the session from {startDate} to {endDate}',
+  futureRunPeriod: {
+    id: 'components.DashboardItemEnrollment.futureRunPeriod',
+    description: 'Text to display the period of a future course run',
+    defaultMessage: 'This session will run from {startDate} to {endDate}',
+  },
+  onGoingEnrolledRunPeriod: {
+    id: 'components.DashboardItemEnrollment.onGoingEnrolledRunPeriod',
+    description: 'Text to display the period of an ongoing course run that user is enrolled to.',
+    defaultMessage: "You are enrolled for this session. It's open from {startDate} to {endDate}",
+  },
+  futureEnrolledRunPeriod: {
+    id: 'components.DashboardItemEnrollment.futureEnrolledRunPeriod',
+    description: 'Text to display the period of a future course run that user is enrolled to.',
+    defaultMessage:
+      'You are enrolled for this session. It will be open from {startDate} to {endDate}',
```

Comment:
Ditto Somewhere else, we are displaying in how many times the course will starts. IMO, it's the more relevant information.

Reply:
I don't know, if i'm a client i like to have a date to log in my calendar. Not to compute "in 14 days", that's mean Wednesday in two week.